tests/test_structs.py

```diff
@@ -745,6 +745,15 @@ def test_segment():
     segment.set_option(option)
     assert segment.has_option()
     assert segment.get_option() == option
+    # Test adding and accessing goals to segments.
```

Comment:
Test adding goals to segments and accessing them.

Reply:
fixed, thanks!